lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -54,6 +54,18 @@ class GoogleCloudOutput < BufferedOutput
     config_param :zone, :string, :default => nil
     config_param :vm_id, :string, :default => nil
 
+    # List of parsed fields whose values are sent as labels instead of
+    # becoming part of the struct payload.
+    #
+    # This is specified as a JSON array where the 'name' is the field name
+    # parsed by fluentd and the 'value' is the name of the label to send.
```

Comment:
(I updated the description of this PR as well so it will be obvious to peope looking at the change history)